api/agentpb/agent.proto

```diff
@@ -117,6 +152,33 @@ message StartActionRequest {
     string dsn = 1;
     string query = 2;
   }
+  // MySQLQueryShowParams describes MySQL SHOW query action parameters.
+  message MySQLQueryShowParams {
+    string dsn = 1;
+    string query = 2;
+  }
+  // MySQLQuerySelectParams describes MySQL SELECT query action parameters.
+  message MySQLQuerySelectParams {
+    string dsn = 1;
+    string query = 2;
+  }
+  // PostgreSQLQueryShowParams describes PostgreSQL SHOW query action parameters.
+  message PostgreSQLQueryShowParams {
+    string dsn = 1;
+  }
+  // PostgreSQLQuerySelectParams describes PostgreSQL SELECT query action parameters.
+  message PostgreSQLQuerySelectParams {
+    string dsn = 1;
+    string query = 2;
+  }
+  // MongoDBQueryGetParameterParams describes MongoDB getParameter query action parameters.
+  message MongoDBQueryGetParameterParams {
+    string dsn = 1;
+  }
```

Comment:
@AlekSi  why here is no query field? How we pass required parameter name?

Reply:
You asked the same question for PostgreSQL above :) There it is always `"*"`. https://docs.mongodb.com/manual/reference/command/getParameter/

Reply:
Yeah, but maybe that is a different case. Well, then we have a bug in validation func.